src/v2/guide/class-and-style.md

```diff
@@ -105,7 +105,7 @@ Si le gustaría activar una clase en la lista de manera condicional, usted puede
 
 Esto siempre aplicará `errorClass`, pero solo aplicará `activeClass` cuando `isActive` sea verdadero.
 
-Sin embaro, esto puede ser un poco verboragico si usted tiene multiple clases condicionales. Por eso también es posible utilizar la sintaxis de objeto dentro de la sintaxis de colección:
+Sin embargo, esto puede ser un poco verboragico si usted tiene multiple clases condicionales. Por eso también es posible utilizar la sintaxis de objeto dentro de la sintaxis de colección:
```

Comment:
Found another typo in here 😅 Can you fix it as well?
```suggestion
Sin embargo, esto puede ser un poco verborrágico si usted tiene multiple clases condicionales. Por eso también es posible utilizar la sintaxis de objeto dentro de la sintaxis de colección:
```

Reply:
We're good to go :rocket: